public/pages/utils/__tests__/constants.ts

```diff
@@ -134,3 +134,171 @@ export const selectedDetectors = [
     lastUpdateTime: 1651818220194,
   },
 ] as DetectorListItem[];
+
+export const anomalyResultSummary = {
+  ok: true,
+  response: {
+    took: 1,
+    timed_out: false,
+    _shards: {
+      total: 1,
+      successful: 1,
+      skipped: 0,
+      failed: 0,
+    },
+    hits: {
+      total: {
+        value: 255,
+        relation: 'eq',
+      },
+      max_score: 0,
+      hits: [
+        {
+          _index: '.opendistro-anomaly-results-history-2022.05.06-1',
+          _id: 'hNX8l4ABuV34PY9I1EAZ',
```

Comment:
nit: given you have the ID hardcoded in the tests a bunch, could be useful to pull this into a constant as well to reference within here + in all of the tests that are using the same ID

Reply:
made the change